Tools/Pipeline/Gtk/Dialogs/TextEditorDialog.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using Gtk;
+
+namespace MonoGame.Tools.Pipeline
+{
+    public partial class TextEditorDialog : Dialog
```

Comment:
That dialog can be used for multiple things. Currently it's used for giving a name to the new folder, later on it can be used with renaming.